src/controls/EK-mapfishprint.js

```diff
@@ -607,7 +607,7 @@ const Mapfishprint = function Mapfishprint(options = {}) {
         } else if (backgroundLayer.getSource() instanceof ImageWMS) { // ----------------------------ol
           url = fetchSourceUrl(backgroundLayer);
         } else {
-          console.warn('Bakgrundslager är av okänd bildtyp: ', backgroundLayer.getSource());
+          console.warn('Bakgrundslager är av okänd bildtyp');
```

Comment:
@Grammostola We made this change as the server crashed with an error at this row as the WMS layer containing css did not have a property.getSource(). We could remove this line from the PR though since it is not linting,. We could gaurd against getSource() not being defined in the catch block to maintain the same functionality.

Reply:
Hi, I see, that sounds good; we can accommodate css-styled layers in a separate PR.